src/dicarlo_lab_to_nwb/conversion/data_locator.py

```diff
@@ -132,3 +132,79 @@ def locate_mworks_processed_file_path(
     assert mworks_processed_file_path.is_file(), f"Mworks file not found: {mworks_processed_file_path}"
 
     return mworks_processed_file_path
+
+
+def locate_session_paths(
```

Comment:
OK, so this is a new file structure. Seems good to have then. I don't understand well what is the output? The docstring says tha it locates the intan file but the output of is a list of Paths.